src/modules/menu/components/LayerDescriptionPopup.vue

```diff
@@ -59,7 +59,12 @@ onMounted(async () => {
 </script>
 
 <template>
-    <ModalWithBackdrop :title="title" :allow-print="true" @close="emit('close', layerId)">
+    <ModalWithBackdrop
+        :title="title"
+        :allow-print="true"
+        :movable="true"
```

Comment:
Small hint, if you want to set a Boolean prop to true, you can just add its name and "voilà"
```
<ModalWithBackdrop
    :title="title"
    allow-print
    movable
    @close="emit('close', layerId)"
>
```